util/common/agent_util_unix.go

```diff
@@ -121,6 +121,26 @@ func InstallAgent(installerFilePath string) error {
 	return err
 }
 
+func StartAgentWithMultiConfig(configOutputPath string, fatalOnFailure bool, ssm bool) error {
+	path := "file:"
+	if ssm {
+		path = "ssm:"
+	}
+	out, err := exec.
+		Command("bash", "-c", "sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a append-config -m ec2 -s -c "+path+configOutputPath).
```

Comment:
I don't know how append config works. I assume if the agent is already running with a config then you run this command it add this config to already running config?